zebra-consensus/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zebra-consensus"
-version = "3.0.0"
+version = "3.0.1"
```

Comment:
> - Added `BlockVerifierService` helper trait for convenience when constraining generic type parameters ([#10010](https://github.com/ZcashFoundation/zebra/pull/10010))

This might also be a minor bump, since adding a new public trait expands the public API and is considered new functionality under SemVer.